lib/tasks/dataverse_migration.rake

```diff
@@ -55,26 +55,26 @@ namespace :migration do
       MigrationLogger.info "Processing the object #{object_id}"
       #reading the metadata file
       metadata_file = Nokogiri::XML(File.open(file))
-      metadata = metadata_file.xpath("//ddi_to_dcterms", MigrationConstants::NS)
+      metadata = metadata_file.xpath("//ddi_to_dcterms", CommonConstants::NS)
```

Comment:
Where does CommonConstants come from, just so I understand the context of this change?